### PR TITLE
[2.13.x] DDF-3941 Add Jetty User Agent to CSRF Filter

### PR DESCRIPTION
**Pull request by duckbyte**
#### What does this PR do?
Adds Jetty user agent to the CSRF filter.

#### Who is reviewing it? 

@blen-desta 
@garrettfreibott 
@peterhuffer 

#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Ask 2 committers to review/merge the PR and tag them here.
@coyotesqrl 
@figliold

#### How should this be tested?

Attempt to make a request with a Jetty user agent. 

#### What are the relevant tickets?
[DDF-3941](https://codice.atlassian.net/browse/DDF-3941)

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: platform/security/filter/security-filter-csrf/src/main/java/org/codice/ddf/security/filter/csrf/CsrfFilter.java [hunk 1]
```diff
@@ -48,6 +48,7 @@
   private static final String APACHE_USER_AGENT = "Apache-CXF/.*";
   private static final String JAVA_CLIENT_USER_AGENT = "Google-HTTP-Java-Client/.*";
   private static final String JAVA_USER_AGENT = "Java/.*";
+  private static final String JETTY_USER_AGENT = "Jetty/.*";
```
Comment (coyotesqrl) [inline platform/security/filter/security-filter-csrf/src/main/java/org/codice/ddf/security/filter/csrf/CsrfFilter.java]: ❓ Is there a good reason not to move these into the blueprint as a second configurable list?

Comment (blen-desta) [reply platform/security/filter/security-filter-csrf/src/main/java/org/codice/ddf/security/filter/csrf/CsrfFilter.java]: There isn't

Comment (ethantmanns) [reply platform/security/filter/security-filter-csrf/src/main/java/org/codice/ddf/security/filter/csrf/CsrfFilter.java]: I'll move it into blueprint

**Comment by blen-desta** [comment]
build now

**Comment by ethantmanns** [comment]
build now

**Comment by blen-desta** [comment]
💯 Hero'd successfully. Verified that ingest, query, and download urls still work.

**Comment by blen-desta** [comment]
build now

**Pull request merged**